src/fiat-connect-client.ts

```diff
@@ -39,19 +39,24 @@ const NETWORK_CHAIN_IDS = {
 }
 const SESSION_DURATION_MS = 14400000 // 4 hours
 
-const fetch = fetchCookie(nodeFetch)
-
 export class FiatConnectClient implements FiatConnectApiClient {
   config: FiatConnectClientConfig
   signingFunction: (message: string) => Promise<string>
   _sessionExpiry?: Date
+  fetchImpl: (url: RequestInfo, init?: RequestInit) => Promise<Response>
 
   constructor(
     config: FiatConnectClientConfig,
     signingFunction: (message: string) => Promise<string>,
+    fetchImpl?: (url: RequestInfo, init?: RequestInit) => Promise<Response>,
```

Comment:
I feel a bit odd leaving the default option as something thats known to be buggy. Will somebody else have to go through the same debugging process we did in order to discover that they need to use this optional param?
We should at least have some comment or recipe explaining when to use this param.